client/state/data-layer/wpcom/concierge/schedules/shifts/from-api.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependenceis
+ */
+import { makeParser } from 'state/data-layer/wpcom-http/utils';
+import responseSchema from './schema';
+
+const transformShift = shift => ( {
+	beginTimestamp: get( shift, 'begin_timestamp' ),
+	endTimestamp: get( shift, 'end_timestamp' ),
+	scheduleId: get( shift, 'schedule_id' ),
+	description: get( shift, 'description' ),
```

Comment:
there's probably no need for any of these `get()`s for two reasons:
  - they aren't actually doing anything since these aren't nested queries. there's no additional safety or any changed behavior. directly accessing `shift.begin_timestamp` is going to be exactly the same as `get( shift, 'begin_timestamp' )`
  - even if they were nested, since we're operating inside of `makeParser` if a nested access throws an error it will immediately fail the parse and switch over into the `onFailure` handler :smile: so we can do `shift.more_data.foo` without concern

Reply:
Nice tip :D I think I'm just too used to apply `lodash.get`.
I've removed it in e3ef256 so it's more eco-friendly now 🌏